<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classesWindow</class>
 <widget class="QMainWindow" name="classesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>دسته ها  - تنظیمات</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/classification.png</normaloff>../icons/classification.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>4</y>
      <width>361</width>
      <height>309</height>
     </rect>
    </property>
    <property name="title">
     <string>تنظیم دسته ها</string>
    </property>
    <widget class="QLabel" name="classLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>دسته ها:</string>
     </property>
    </widget>
    <widget class="ExtendedComboBox" name="classComboBox">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>27</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="activeClassCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>29</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>فعال</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="settingGroupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>90</y>
       <width>341</width>
       <height>209</height>
      </rect>
     </property>
     <property name="title">
      <string>تنظیمات</string>
     </property>
     <widget class="QLineEdit" name="folderPathLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>52</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="classAparatCategoryLabel">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>112</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>دسته بندی آپارات:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classAparatCategoryLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>112</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="folderPathLabel">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>52</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>آدرس پوشه:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>172</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="usenameLabel">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>142</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>نام کاربری کانال:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>142</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="classNameLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>22</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="classPlaylistNameLabel">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>82</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>لیست پخش:</string>
      </property>
     </widget>
     <widget class="QLabel" name="classNameLabel">
      <property name="geometry">
       <rect>
        <x>281</x>
        <y>22</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>نام دسته:</string>
      </property>
     </widget>
     <widget class="QLabel" name="passwordLabel">
      <property name="geometry">
       <rect>
        <x>217</x>
        <y>172</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>رمز عبور کانال:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="classPlaylistNameLineEdit">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>82</y>
        <width>223</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="newClassPushButton">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>60</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>دسته جدید</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteClassPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>60</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>حذف دسته</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editClassPushButton">
     <property name="geometry">
      <rect>
       <x>203</x>
       <y>60</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ویرایش دسته</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExtendedComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">source/extended/ExtendedComboBox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>classComboBox</tabstop>
  <tabstop>activeClassCheckBox</tabstop>
  <tabstop>editClassPushButton</tabstop>
  <tabstop>newClassPushButton</tabstop>
  <tabstop>deleteClassPushButton</tabstop>
  <tabstop>classNameLineEdit</tabstop>
  <tabstop>folderPathLineEdit</tabstop>
  <tabstop>classPlaylistNameLineEdit</tabstop>
  <tabstop>classAparatCategoryLineEdit</tabstop>
  <tabstop>usernameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
